test/test_pebble.py

```diff
@@ -2886,13 +2921,15 @@ def test_get_checks_all(self, client: MockClient):
                     'name': 'chk1',
                     'status': 'up',
                     'threshold': 2,
+                    'change-id': '1',
                 },
                 {
                     'name': 'chk2',
                     'level': 'alive',
                     'status': 'down',
                     'failures': 5,
                     'threshold': 3,
+                    'change-id': '3',
```

Comment:
Are these ids important?
The test doesn't seem to validate these values.

Reply:
It's important that they aren't the empty string or missing (or None, although that shouldn't happen). I can add in validation that they are as expected, although that is covered by other tests.